src/main/drivers/serial_uart_stm32f4xx.c

```diff
@@ -262,7 +264,7 @@ void uartIrqHandler(uartPort_t *s)
 
 static void handleUsartTxDma(uartPort_t *s)
 {
-    // DMA_Cmd(s->txDMAStream, DISABLE);
+    DMA_Cmd(s->txDMAStream, DISABLE);
```

Comment:
It works without disabling it in `handleUsartTxDma` ...

Will do some counting on number of fresh starts and interrupts...